requirements.txt

```diff
@@ -1,5 +1,6 @@
 knackpy==1.0.*
-pandas==1.5.*
+mstrio-py==11.3.4.*
+pandas
```

Comment:
did you run into an issue with the pinned version for `pandas` here?

Reply:
Yeah mstrio-py is verry picky with it's dependencies. Probably not necessary to include pandas in requirements.txt because it's included with mstrio-py.